onnx_caffe2/backend.py

```diff
@@ -612,21 +653,15 @@ def _common_onnx_node_to_caffe2_op(cls, onnx_node, opset_version):
         If you're writing a custom translator, consider calling this first,
         and then fixing things up further.
         """
+        cls.validate(onnx_node, opset_version)
```

Comment:
It's better to insert the validation into `_onnx_node_to_caffe2_op` which will then cover both special and non-special translations.